src/security.cs

```diff
@@ -337,178 +337,220 @@ interface KeysAccessible {
 		IntPtr WhenPasscodeSetThisDeviceOnly { get; }
 	}
 
-	[Static][Internal]
```

Comment:
Yea so `SecAttributeKey` and friends were made `[Internal]` because they have a strongly typed counterpart. See https://github.com/xamarin/xamarin-macios/blob/master/src/Security/Items.cs#L771-L782 for instance.

However given https://bugzilla.xamarin.com/show_bug.cgi?id=34135 it seems like it's useful to be able to access those constants.

I would add `[EditorBrowsable (EditorBrowsableState.Never)]` to the interface declaration though so those keys don't show up in intellisense and we favor the strongly typed APIs.

Reply:
I would go `[EditorBrowsable (EditorBrowsableState.Advanced)]` instead so they can be discovered using the editor this way.

Reply:
Yea I wrote `EditorBrowsableState.Never` initially but then corrected by `EditorBrowsableState.Advanced`.

Reply:
Done in https://github.com/xamarin/xamarin-macios/pull/3350/commits/60feaf5e748d4208e92c950f5fc5c9b78d6daba3